### PR TITLE
chore: add game design 101 in youtube/game-development

### PR DESCRIPTION
**Pull request by ojasaklechayt**
## Fixes Issue
#1127

## Changes proposed
Added youtube playlist for the game development.

## Screenshots
image

## Note to reviewers
I have asked to assign this to me in the issue itself.

### CODE REVIEW

**Review by rupali-codes — CHANGES_REQUESTED** [review]
could you please format the code?

**Comment by ojasaklechayt** [comment]
> could you please format the code?

Where should I make the changes ?

File: database/youtube/game-development.json [hunk 1]
```diff
@@ -1 +1,8 @@
-[]
+[{
+    "name": "Game Design 101",
+    "description": "Videos that look at the fundamental lessons of game design, applicable to pretty much every type of game. It is playlist created by Mark Brown a well know game developer",
```
Comment (CBID2) [inline database/youtube/game-development.json:3]: [Proofreading]
```suggestion
    "description": "This playlist consists of videos where Mark Brown, a well-known game developer, teaches you the fundamental lessons of game design. These videos are applicable to pretty much every type of game.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ojasaklechayt! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly before making a pull request. Other than that, you're off to a great start! :)

**Comment by ojasaklechayt** [comment]
> Hi @ojasaklechayt! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly before making a pull request. Other than that, you're off to a great start! :)

I have made the changes 👍🏽

**Review by rupali-codes — COMMENTED** [review]
could you please format the code _(the braces)_?

**Comment by ojasaklechayt** [comment]
> could you please format the code _(the braces)_?

Done Ma'am

**Review by rupali-codes — APPROVED** [review]

**Review by CBID2 — APPROVED** [review]
Lgtm

**Pull request merged**